eng/pipelines/partner-release.yml

```diff
@@ -110,8 +110,8 @@ extends:
               targetPath: '$(Pipeline.Workspace)/packages-esrp-flattened'
 
           pool:
-            name: azsdk-pool-mms-win-2022-general
-            image: azsdk-pool-mms-win-2022-1espt
+            name: $(WINDOWSPOOL)
```

Comment:
IIRC there were some issues with deployment jobs not being able to use variables so do be sure to test the deployment job with this change.

Reply:
yeah, that [kind of looks like what's going on](https://dev.azure.com/azure-sdk/internal/_build/results?buildId=4630247&view=results) with an outstanding lack of errors messages or any indication that's the case